### ISSUE TITLE
Focused cell is not visible enough when it is also the *selected* cell

### ISSUE THREAD

**Issue by ExtAnimal** [bryntum/support#9906]
[Forum post](https://forum.bryntum.com/viewtopic.php?t=30253)

The focus styling result confusing. We can spot two different issues here:

When setting focus on picked date cell, the picked date cell styling is not covering the case that the cell is also being focused. It remains with the same styling independently of being focused. This can lead the user to think that the focus has been lost when placed in the picked cell. (Attached photo: focusIsPlacedOnPickedCellButNoBorderIsShowingFocus.png)
This will be breaking WACG rule 2.4.7 Focus not visible: https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-focus-visible.html

When moving the focus between (not selected) date cells by arrow keys, in the date table, the date cell presents an orange border outline which represents that cell is being focused. The problems is spotted when pressed shift+tab, this will move focus to the toolbar buttons but the style of the last focused date cell still persistent. This can leads to confusion due to not knowing in which element the focus could be. (Issue show in video: dateCellStillShowingOrangeBorder)

Steps to reproduce in bryntum examples:
https://bryntum.com/products/calendar/examples/print/

Problem:
Navigate to the calendar date picker using keyboard.
Move focus with arrow keys between the date cells.
Press Enter to select the date cell you are focusing.
Focus seems "lost" because there is no visual indication that the focus is on the selected date.

Problem:
Navigate to the calendar date picker using keyboard.
Move focus with arrow keys between the date cells, the focused cells that is not selected has an orange border indicating that is focused
Press shift + tab to move focus to toolbar buttons of the date picker widget.
You will notice that the focus is not visible for the header year button and the currently selected date cell.
![focusIsPlacedOnPickedCellButNoBorderIsShowingFocus.png](https://forum.bryntum.com/download/file.php?id=30494)
![visualFocusOnLastCellWhenFocusIsOnToolbar.png](https://forum.bryntum.com/download/file.php?id=30493)

**Comment by ExtAnimal** [comment]
CSS change to fix the border colour.

<img width="1420" alt="Screenshot 2024-08-28 at 15 09 41" src="https://github.com/user-attachments/assets/92cd2025-c7cf-45c1-bfdb-a67d50b98249">

It's still not very visible. For an accessibility aid, you need the eyes of a hawk to see that border.

<img width="240" alt="Screenshot 2024-08-28 at 15 11 00" src="https://github.com/user-attachments/assets/d5dc19ee-8fd4-418e-b687-9acdffea5d61">
